### PR TITLE
Make the probe list configurable

### PR DESCRIPTION
**Pull request by greg-hellings**
Addresses #2

### CODE REVIEW

**Comment by greg-hellings** [comment]
@ascerra - please be sure to test this and see it addresses your issue

**Comment by ascerra** [comment]
@greg-hellings this change has fixed the issue I was facing. I am now able to pass the necessary IPs as part of an ansible playbook to probe the hosts. Thank you for the quick fix!

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Currently the following variables are supported:
 
 ### General
 
+* `gluster_gdeploy_peers` - A list of hostnames, IP addreses, or similar network
+  addresses that should be probled into the Gluster cluster. This defaults to all
```
Comment (seandst) [inline README.md]: Out of curiosity, how do you proble a system? Can you also droble a system? :)

Comment (robled) [reply README.md]: You can definitely (p)robled a system.

Comment (greg-hellings) [reply README.md]: Oy. #4

**Pull request merged**